### PR TITLE
fix: UI front fixes

### PR DESCRIPTION
**Pull request by clegirar**
- [x] Re-enable Reactions on message ( #4275 )
- [x] Move `Appearence` button in devtools (we don't support theming for the moment, #4090 #4246 #4258)
- [x] Quote of the day (when no results from search in Home, #4288 )
- [x] Emoji message is cut off ( #4287 )
- [x] In delete account screen, the input value should be the translation of "delete" in the current language and not directly `delete` word
- [x] Change multimember conversation avatar picture upload logic (#4225)
- [x] Fixed the camera that's open twice in the "+" Menu in conversation footer ( #4245 )

Signed-off-by: clegirar <clemntgirard@gmail.com>

### CODE REVIEW

File: js/packages/components/chat/message/Reactions.tsx [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+			) || [],
+		)
+	}, [cid, client, currentEmoji, members])
+
+	useEffect(() => {
+		setCurrentEmoji(emoji)
+	}, [emoji])
+
+	useEffect(() => {
+		getReactionList().then()
```
Comment (n0izn0iz) [inline js/packages/components/chat/message/Reactions.tsx:86]: no need to call `then`

File: js/packages/components/chat/message/Reactions.tsx [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+				})
+			},
+			(dataSourceCords || []).length === reactions.length ? 0 : 50,
+		)
+		scrollViewRef.current?.scrollTo({
+			x: dataSourceCords ? dataSourceCords[index] : 0,
+			y: 0,
+			animated: true,
+		})
+	}, [currentEmoji, dataSourceCords, reactions, scrollViewRef])
```
Comment (n0izn0iz) [inline js/packages/components/chat/message/Reactions.tsx:106]: please add comment explaining why we need the `setTimeout`

**Review by n0izn0iz — APPROVED** [review]
LGTM except comments

File: js/packages/components/chat/message/Reactions.tsx (hunk 2 above)
Comment (clegirar) [reply js/packages/components/chat/message/Reactions.tsx:106]: I don't know why i just get the last version of this file to re-enable the feature

**Pull request merged**